examples/ads.amp.html

```diff
@@ -1879,6 +1880,13 @@ <h2>SAS CI 360 Match</h2>
       data-tags='{"CAT":"SYLVESTER","POSITION":"TOP"}'>
   </amp-ad>
 
+  <h2>Seeding Alliance</h2>
+  <amp-ad 
+    height="250"
+    type="seedingalliance"
+    data-url="mockup.seeding-alliance.de">
```

Comment:
If your backend allows, it would be nice to include the https:// protocol as well if that works. Let me know if you do not, and I can approve.

Reply:
That's not a problem, I added the protocol.